encrypted-media-respec.html

```diff
@@ -2331,15 +2365,16 @@
 
         <section id="queue-waitingforkey">
           <h4>Queue a "waitingforkey" Event</h4>
-          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt the current block.
+          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt an encrypted block.
             It should only be called when the <a def-id="htmlmediaelement"></a> object is <a def-id="potentially-playing"></a> and its <a def-id="readystate"></a> is equal to <a def-id="have-future-data"></a> or greater.  
```

Comment:
@mwatson2, I was replying to @cpearce's comment, not the actual text (even though that is apparently how GitHub shows the comment).

Thinkin about it more, it's not clear that `HAVE_CURRENT_DATA` means the data has actually been decoded or displayed. If not, perhaps `HAVE_CURRENT_DATA` is the minimum state.